homeassistant/components/climate/nest.py

```diff
@@ -62,13 +63,22 @@ def unit_of_measurement(self):
         return TEMP_CELSIUS
 
     @property
+    def state(self):
```

Comment:
Do not overwrite state. That's for the Climate ABC to decide (it currently returns `self.operation_mode`)